homework_2/multiply.py

```diff
@@ -0,0 +1,7 @@
+#  Write a function that will receive 2 numbers as input and it should return the multiplication of these 2 numbers.
+
+
+def mult_two(arg1, arg2):
+    if not isinstance(arg1, (int, float)) or not isinstance(arg2, (int, float)):
```

Comment:
I would suggest to avoid not conditions since they are hard to read, If you see condition too dificult, move it to separate variable:

direct condition is easier to read, and readable variable suggests what happens here
```
    both_numbers =  isinstance(arg1, (int, float)) and isinstance(arg2, (int, float))

    if notboth_numbers:
        ...

```